java/runtime/src/main/java/io/ray/runtime/metric/RayMetrics.java

```diff
@@ -0,0 +1,148 @@
+package io.ray.runtime.metric;
+
+import com.google.common.base.Preconditions;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The entry of metrics for easy use.
+ */
+public final class RayMetrics {
```

Comment:
Just name this class `Metrics`, we don't use `RayXxx` now.

Reply:
OK.